x-pack/plugin/sql/qa/server/src/main/resources/date.csv-spec

```diff
@@ -38,7 +38,7 @@ SELECT TRUNCATE(YEAR(TODAY() - INTERVAL 50 YEARS) / 1000) AS result;
 
 
 currentDateFilter
-SELECT first_name FROM test_emp WHERE hire_date > CURRENT_DATE() - INTERVAL 35 YEARS ORDER BY first_name ASC LIMIT 10;
+SELECT first_name FROM test_emp WHERE hire_date > CURRENT_DATE() - INTERVAL 44 YEARS ORDER BY first_name ASC LIMIT 10;
```

Comment:
Just as it happens an entry in the dataset failed this condition today (21/11/1985) - I've updated the filter to remain the same for another 9 years :)